index.bs

```diff
@@ -674,23 +723,22 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     <dt>If any |authenticator| indicates success,</dt>
     <dd>
-        1. [=set/Remove=] |authenticator| from |issuedRequests|.
-        2. Let |value| be a new {{AuthenticationAssertion}} object associated with |global| whose fields are:
-            : {{AuthenticationAssertion/credential}}
-            :: A new {{ScopedCredential}} object associated with |global| whose fields are:
-                1. {{ScopedCredential/type}} whose value is the {{ScopedCredentialType}} representing this [=scoped 
-                    credential=]'s type.
-                1. {{ScopedCredential/id}} whose value is a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=],
-                    containing the bytes of the credential ID returned from the successful [=authenticatorGetAssertion=]
-                    operation.
-            : {{AuthenticationAssertion/clientDataJSON}}
-            :: A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|
-            : {{AuthenticationAssertion/authenticatorData}}
-            :: A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of the returned
-                {{authenticatorData}}
-            : {{AuthenticationAssertion/signature}}
-            :: A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of the returned
-                {{signature}}
+        1.  [=set/Remove=] |authenticator| from |issuedRequests|.
+        2.  Let |value| be a new {{ScopedCredential}} associated with |global| whose fields are:
+
+            :   {{ScopedCredential/[[identifier]]}}
+            ::  A new {{ArrayBuffer}} containing the bytes of the credential ID returned from the successful
```

Comment:
Thanks! Fixed in https://github.com/w3c/webauthn/pull/384/commits/fedccedb468282665936264105fac6ee09e577cf.